.github/workflows/swift.yml

```diff
@@ -37,8 +37,8 @@ jobs:
     steps:
       - uses: compnerd/gha-setup-swift@main
         with:
-          branch: swift-5.10-release
-          tag: 5.10-RELEASE
+          branch: swift-6.0.2-release
```

Comment:
@migueldeicaza I thought I'd try bumping the version right up to 6.0.2 here to see if the Windows compiler's argument pack support was fleshed out. The tests pass so I guess that it is.

I guess it might be worth setting the swift-tools-version to 6.0 if we truly do need Swift 6 on Windows - but then we'd have to add an explicit opt-out of the Swift 6 language model for now, to avoid having to deal with that whole can of worms...